src/components/ArticleCard/index.tsx

```diff
@@ -32,9 +32,21 @@ const ArticleCard = ({
   const { t } = useTranslation(translation);
   const history = useHistory();
 
-  const clickHandler = (url: string) => {
+  const clickHandler = (
+    e: React.MouseEvent<HTMLElement>,
+    cardRoute: string
+  ) => {
+    const target = e.target as Element;
+    if (target.getAttribute('data-testid') === 'tag') {
+      if (type === 'Section 508') {
+        history.push('/help/section-508');
+      } else {
+        history.push('/help/it-goverance');
+      }
+      return;
+    }
```

Comment:
The `<Tag>` component below needs to be updated to `<HelpTag type={tag} />` to use the shared components. So the clickHandler could be simplified to:
`const target = e.target as Element;`
`if (isLink && target.getAttribute('data-testid') !== 'tag') {`
`history.push(url);`
 `}`